Stream Browser Error

In the Stream Browser  press the filter then `Feed = some feed`
press OK, 
press the filter and select `Feed = None`


You will get then an error

```
ResponseException{method='POST', url='https://xxxk/api/meta/v1/find', json='{"code":500,"message":"Unable to find doc with reference 'DocRef{type='', uuid='', name='None'}' for term: Feed is None","details":"java.lang.RuntimeException: Unable to find doc with reference 'DocRef{type='', uuid='', name='None'}' for term: Feed is None"}', code='500', message='Unable to find doc with reference 'DocRef{type='', uuid='', name='None'}' for term: Feed is None', details='java.lang.RuntimeException: Unable to find doc with reference 'DocRef{type='', uuid='', name='None'}' for term: Feed is None', responseKeyValues='code: 500, message: Unable to find doc with reference 'DocRef{type='', uuid='', name='None'}' for term: Feed is None, details: java.lang.RuntimeException: Unable to find doc with reference 'DocRef{type='', uuid='', name='None'}' for term: Feed is None'}
	at Unknown.Hh(stroom-0.js)
	at Unknown.Uh(stroom-0.js)
	at Unknown.new Pof(stroom-0.js)
	at Unknown.wof(stroom-0.js)
	at Unknown.Dof(stroom-0.js)
	at Unknown.aqd(stroom-0.js)
	at Unknown.dw(stroom-0.js)
	at Unknown.rw(stroom-0.js)
	at Unknown.eval(stroom-0.js)
	at Unknown.ij(stroom-0.js)
	at Unknown.lj(stroom-0.js)
	at Unknown.eval(stroom-0.js)
Caused by: org.fusesource.restygwt.client.FailedResponseException: 
	at Unknown.Rh(stroom-0.js)
	at Unknown.Th(stroom-0.js)
	at Unknown.hqd(stroom-0.js)
	at Unknown.new iqd(stroom-0.js)
	at Unknown.aqd(stroom-0.js)
	at Unknown.dw(stroom-0.js)
	at Unknown.rw(stroom-0.js)
	at Unknown.eval(stroom-0.js)
	at Unknown.ij(stroom-0.js)
	at Unknown.lj(stroom-0.js)
	at Unknown.eval(stroom-0.js)
```

--- a/stroom-core-client/src/main/java/stroom/explorer/client/presenter/ExplorerDropDownTreePresenter.java
+++ b/stroom-core-client/src/main/java/stroom/explorer/client/presenter/ExplorerDropDownTreePresenter.java
@@ -202,7 +202,7 @@ class ExplorerDropDownTreePresenter
     }
 
     private static ExplorerNode resolve(final ExplorerNode selection) {
-        if (selection == ExplorerTreeModel.NULL_SELECTION) {
+        if (ExplorerTreeModel.NULL_SELECTION.equals(selection)) {
             return null;
         }
 

--- a/unreleased_changes/20230911_152146_161__3738.md
+++ b/unreleased_changes/20230911_152146_161__3738.md
@@ -1,0 +1,24 @@
+* Issue **#3738** : Fix error when selecting Feed is None in stream filter.
+
+
+```sh
+# ********************************************************************************
+# Issue title: Stream Browser Error
+# Issue link:  https://github.com/gchq/stroom/issues/3738
+# ********************************************************************************
+
+# ONLY the top line will be included as a change entry in the CHANGELOG.
+# The entry should be in GitHub flavour markdown and should be written on a SINGLE
+# line with no hard breaks. You can have multiple change files for a single GitHub issue.
+# The  entry should be written in the imperative mood, i.e. 'Fix nasty bug' rather than
+# 'Fixed nasty bug'.
+#
+# Examples of acceptable entries are:
+#
+#
+# * Issue **123** : Fix bug with an associated GitHub issue in this repository
+#
+# * Issue **namespace/other-repo#456** : Fix bug with an associated GitHub issue in another repository
+#
+# * Fix bug with no associated GitHub issue.
+```

